hazelcast/serialization/portable/writer.py

```diff
@@ -145,6 +194,23 @@ def write_utf(self, field_name, value):
     def write_utf_array(self, field_name, values):
         self.write_string_array(field_name, values)
 
+    def write_nullable_field(self, field_name: str, field_type: FieldType, value, func):
+        self._set_position(field_name, field_type)
+        is_nil = value is None
```

Comment:
is_none or is_null would be more appropriate for python

Reply:
Fixed at https://github.com/hazelcast/hazelcast-python-client/pull/569/commits/6b34418b0dd46c23cb7a5ed87e84d83865ce27e1.